app/components/candidate_interface/application_dashboard_course_choices_component.rb

```diff
@@ -143,7 +143,12 @@ def status_row(application_choice)
       if @show_status
         {
           key: 'Status',
-          value: render(ApplicationStatusTagComponent.new(application_choice:)),
+          value: render(
+            ApplicationStatusTagComponent.new(
+              application_choice:,
+              supplementary_statuses: supplementary_statuses_for(application_choice:),
```

Comment:
On seconds thoughts I reckon the handling of supplementary statuses should probably be handled in the component itself rather than passed in. `supplementary_statuses_for` should probably be a method on the `ApplicationChoice` model rather than a helper method. Any thoughts?